Firestore/core/test/unit/model/object_value_test.cc

```diff
@@ -35,43 +35,30 @@ using testutil::DbId;
 using testutil::Field;
 using testutil::Map;
 using testutil::Value;
+using testutil::WrapObject;
 
 class ObjectValueTest : public ::testing::Test {
- public:
-  template <typename T>
-  google_firestore_v1_Value Wrap(T input) {
-    model::FieldValue fv = Value(input);
-    return serializer.EncodeFieldValue(fv);
-  }
-
-  template <typename... Args>
-  MutableObjectValue WrapObject(Args... key_value_pairs) {
-    FieldValue fv = testutil::WrapObject((key_value_pairs)...);
-    return MutableObjectValue{serializer.EncodeFieldValue(fv)};
-  }
-
  private:
   remote::Serializer serializer{DbId()};
 };
 
 TEST_F(ObjectValueTest, ExtractsFields) {
-  MutableObjectValue value =
-      WrapObject("foo", Map("a", 1, "b", true, "c", "string"));
+  ObjectValue value = WrapObject("foo", Map("a", 1, "b", true, "c", "string"));
 
   ASSERT_EQ(google_firestore_v1_Value_map_value_tag,
             value.Get(Field("foo"))->which_value_type);
 
-  EXPECT_TRUE(Wrap(1) == *value.Get(Field("foo.a")));
-  EXPECT_TRUE(Wrap(true) == *value.Get(Field("foo.b")));
-  EXPECT_TRUE(Wrap("string") == *value.Get(Field("foo.c")));
+  EXPECT_TRUE(Value(1) == *value.Get(Field("foo.a")));
```

Comment:
Nit: looks like these checks could use `EXPECT_EQ` instead. See go/c-readability-advice#expect-that-vs-other-expect- (I know it's a preexisting issue)

Reply:
Fixed. I wasn't able to EXPECT_EQ at first because `operator==` was declared in the wrong scope.